cypress/fixtures/modules/securestorage.json

```diff
@@ -272,19 +272,130 @@
     },
     "VALUE_TTL": "authTestTokenValueTTL",
     "SETFORAPP_DEVICE": {
-        "appId": "refui",
+        "appId": "boltextn",
         "scope": "device",
         "key": "authTestTokenDevice",
         "value": "VGhpcyBub3QgYSByZWFsIHRva2VuLgo="
     },
+    "SETFORAPP_INVALID_SCOPE": {
+        "appId": "boltextn",
```

Comment:
boltextn is ripple specific. this cannot go in FCS

Reply:
updated to "comcast.test.firecert" and moved to configModule